src/graphql/helpers.ts

```diff
@@ -110,6 +110,21 @@ export function buildWhereQuery(fields, alias, where) {
       params.push(fieldNotIn);
     }
 
+    const fieldChoiceMatch = where[`${field}_match`] || [];
+    if (fieldChoiceMatch.length > 0 && alias === 'v') {
+      params.push(fieldChoiceMatch);
+      const keys: string = fieldChoiceMatch.reduce((acc: string, val: string, i: number) => {
+        const isLast = i === fieldChoiceMatch.length - 1;
+
+        const param = `'$."${val}"'`;
+        params.push(param);
+
+        acc += `?${isLast ? '' : ', '}`;
+        return acc;
+      }, '');
+      query += `AND (${alias}.${field} IN (?) OR JSON_CONTAINS_PATH(${alias}.${field}, 'all', ${keys})) `;
```

Comment:
Wouldn't this be exposed to SQL injection? I don't see how value get parsed on the JSON_CONTAINS_PATH query, someone may bring something like this as value `")); TRUNCATE proposals;"` and delete proposals table isn't?

Reply:
Some input from GPT-4
```
The code does risk SQL injection through the JSON_CONTAINS_PATH part. Although it uses parameterized queries, the keys variable is directly interpolated into the SQL string. This can lead to SQL injection if these values come from an untrusted source like user input. It's safer to use parameter binding for JSON_CONTAINS_PATH, if possible with your SQL library.
```